.github/workflows/e2e-test.yml

```
@@ -180,10 +220,10 @@ jobs:
           cache_enabled: "true"
           test_label: "E2ECritical"
       fail-fast: false
-    name: End to End Critical Scenario Multi User Tests - K8s ${{ matrix.k8s_version }} cacheEnabled=${{ matrix.cache_enabled }} multiUser=${{ matrix.multi_user }} artifactProxy=${{ matrix.artifact_proxy }}
+    name: End to End Critical Scenario Multi User Tests - K8s ${{ matrix.k8s_version }} cacheEnabled=${{ matrix.cache_enabled }} multiUser=${{ matrix.multi_user }} argoVersion=${{ matrix.argo_version}} proxy=${{ matrix.proxy}} storage=${{ matrix.storage }} artifactProxy=${{ matrix.artifact_proxy }}
```

actionlint (1.7.11)

[error] property "argo_version" is not defined in object type {artifact_proxy: bool; cache_enabled: bool; k8s_version: string; multi_user: bool; test_label: string}

(expression)